KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_SKIPMONO='${E("KOREBUILD_TEST_SKIPMONO")}'
+default TEST_ARTIFACT_DIR='${Path.Combine(Directory.GetCurrentDirectory(), "artifacts", "tests")}'
```

Comment:
This would affect any test that attempts to determine paths relative to the test publish directory. For instance https://github.com/aspnet/Localization/blob/dev/test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs#L16.

Could you ready follow up PRs to make sure your changes work with this?